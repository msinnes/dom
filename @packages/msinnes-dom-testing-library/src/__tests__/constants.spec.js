import {
  WINDOW_EVENTS,
  FORM_EVENTS,
  KEYBOARD_EVENTS,
  MOUSE_EVENTS,
  MEDIA_EVENTS,
} from '../constants';

describe('constants', () => {
  describe('window events', () => {
    it('should match the snapshot', () => {
      expect(WINDOW_EVENTS).toMatchSnapshot();
    });
  });

  describe('form events', () => {
    it('should match the snapshot', () => {
      expect(FORM_EVENTS).toMatchSnapshot();
    });
  });

  describe('keyboard events', () => {
    it('should match the snapshot', () => {
      expect(KEYBOARD_EVENTS).toMatchSnapshot();
    });
  });

  describe('mouse events', () => {
    it('should match the snapshot', () => {
      expect(MOUSE_EVENTS).toMatchSnapshot();
    });
  });

  describe('media events', () => {
    it('should match the snapshot', () => {
      expect(MEDIA_EVENTS).toMatchSnapshot();
    });
  });
});