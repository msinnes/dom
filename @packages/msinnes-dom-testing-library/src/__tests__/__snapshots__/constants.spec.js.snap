// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants form events should match the snapshot 1`] = `
[
  "onblur",
  "onchange",
  "oncontextmenu",
  "onfocus",
  "onformchange",
  "onforminput",
  "oninput",
  "oninvalid",
  "onreset",
  "onselect",
  "onsubmit",
]
`;

exports[`constants keyboard events should match the snapshot 1`] = `
[
  "onkeydown",
  "onkeypress",
  "onkeyup",
]
`;

exports[`constants media events should match the snapshot 1`] = `
[
  "onabort",
  "oncanplay",
  "oncanplaythrough",
  "ondurationchange",
  "onemptied",
  "onended",
  "onerror",
  "onloadeddata",
  "onloadedmetadata",
  "onloadstart",
  "onpause",
  "onplay",
  "onplaying",
  "onprogress",
  "onratechange",
  "onreadystatechange",
  "onseeked",
  "onseeking",
  "onstalled",
  "onsuspend",
  "ontimeupdate",
  "onvolumechange",
  "onwaiting",
]
`;

exports[`constants mouse events should match the snapshot 1`] = `
[
  "onclick",
  "ondblclick",
  "ondrag",
  "ondragend",
  "ondragenter",
  "ondragleave",
  "ondragover",
  "ondragstart",
  "ondrop",
  "onmousedown",
  "onmousemove",
  "onmouseout",
  "onmouseover",
  "onmouseup",
  "onmousewheel",
  "onscroll",
]
`;

exports[`constants window events should match the snapshot 1`] = `
[
  "onafterprint",
  "onbeforeprint",
  "onbeforeunload",
  "onerror",
  "onhaschange",
  "onload",
  "onmessage",
  "onoffline",
  "onpagehide",
  "onpageshow",
  "onpopstate",
  "onredo",
  "onresize",
  "onstorage",
  "onundo",
  "onunload",
]
`;
